plugins/reach/lib/api/data/vendorCatalogItem/KalturaVendorConfigSchedulingLiveCaptionCatalogItem.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * @package plugins.reach
+ * @subpackage api.objects
+ */
+class KalturaVendorConfigSchedulingLiveCaptionCatalogItem extends KalturaVendorLiveCaptionCatalogItem
```

Comment:
I haven't read this entire PR 
But why isn't this a config param on the KalturaVendorLiveCaptionCatalogItem? what is there a need to create an entirely new catalog item type for this?

Reply:
@yossipapi  addressed, please review